trulens/nn/attribution.py

```diff
@@ -54,7 +67,9 @@ class AttributionMethod(AbstractBaseClass):
     """
 
     @abstractmethod
-    def __init__(self, model: ModelWrapper, *args, **kwargs):
+    def __init__(
+        self, model: ModelWrapper, doi_per_batch=None, *args, **kwargs
```

Comment:
nit: batch_per_doi/rebatch_per_doi/rebatch_doi/rebatch_for_doi, etc seems like a more accurate names

Reply:
actually, I see that effective_batch_size is not yet used, so I'm not exactly sure the end product of what this parameter will do. either way, maybe add a docstring on that

Reply:
Accidental inclusion. Removing from this PR.